src/index.js

```diff
@@ -31,21 +29,69 @@ const configI18n = userSettings => {
     i18n.changeLanguage(uiLocale);
 };
 
-const init = () => {
+const injectScriptsIntoBody = scriptPrefix => {
```

Comment:
I suggest bundling these artifacts. The core-resources will be deprecated in 2.32 and removed in 2.33.

Do we need jquery still?

Reply:
The plugins (`eventreport`, `eventchart`, `reporttable`, and `maps`) are not yet available as published libraries. So that will have to come later.

As far as jQuery, for now, we could get those from another CDN.

I'm not sure we need to include the babel-polyfill actually. Doesn't CRA bundle all the needed polyfills in the build?

Reply:
> I suggest bundling these artifacts. The core-resources will be deprecated in 2.32 and removed in 2.33.

The plugins (eventreport, eventchart, reporttable, and maps) are not yet available as published libraries. So that will have to come later.

As far as jQuery, for now, we could get those from another CDN. jQuery is unfortunately needed by the older plugins.

I'm not sure we need to include the babel-polyfill actually. Doesn't CRA bundle all the needed polyfills in the build?

Reply:
>The plugins (eventreport, eventchart, reporttable, and maps) are not yet available as published libraries. So that will have to come later.

That's fine, they look like they are linked to the app, not to core-resource. Those are the ones we need to get rid of first and foremost. 

CRA should do nicely unless there is code which uses something special.

External CDNs is not a good idea. Some of the DHIS2 instances are on isolated networks which may or may not have external internet access. Better to bundle it with the app or the plugin. If it's required it _is_ a dependency.

Reply:
Yeah

> Relying on an external CDN is not a good idea

Yeah, I realized that right after I wrote it - thought I changed my comment, but apparently not in time. I'll look into bundling jQuery and see what is needed from babel-polyfill (Bjørn says Maps needs it)

Reply:
> CRA polyfill should do nicely unless there is code which uses something special.

I think we need to include `import 'react-app-polyfill/ie11';` from [CRA](https://github.com/facebook/create-react-app/tree/master/packages/react-app-polyfill) to support ie11?

> see what is needed from babel-polyfill (Bjørn says Maps needs it)

(note: maps doesn't use CRA currently)

Reply:
> I think we need to include `import 'react-app-polyfill/ie11';` from [CRA](https://github.com/facebook/create-react-app/tree/master/packages/react-app-polyfill) to support ie11?

Oh, shoot. Yeah they changed that in CRA2.0. You're correct.